frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -311,11 +321,11 @@
 
       <div>
         <p class="text-2xl mb-4">
-          {$t('project_page.members')}
+          {$t('project_page.members.title')}
         </p>
 
-        <BadgeList>
-          {#each project.users as member}
+        <BadgeList grid>
```

Comment:
```suggestion
        <BadgeList grid={showMembers.length > TRUNCATED_MEMBER_COUNT}>
```
makes it only a grid if the number visible is greater than the truncated amount. Meaning when it's collapsed it will just be inline. Here's a gif in action
![firefox_ClAEqww56Q](https://github.com/sillsdev/languageforge-lexbox/assets/4575355/05a769f8-ad74-4265-81ce-1f5b8e6e7a87)
Here's how it looks with just 2 members, not a grid, exactly how it looks today
![image](https://github.com/sillsdev/languageforge-lexbox/assets/4575355/61b224bd-6ecb-4888-8ef6-e201992d9834)
I kinda like the organic feel of our member list today and was trying to preserve that.

What do you think?